oat-sa/extension-tao-delivery
# Feature/tr 858/rtl delivery header

**Related to:** https://oat-sa.atlassian.net/browse/TR-858

**Changes:**
- added styles and initial code o implement RTL feature for felivery headers

**How to check:**
- open delivery index page and/or launch delivery
- ensure header elements order and position is reverted

**Note:**
- PR requires feature implemented in [TR-857](https://oat-sa.atlassian.net/browse/TR-857)

**Requires:**
 - [ ] mentioned in **Note** section
